samples/stock/demo/compare-etf/demo.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/stock/modules/accessibility.js"></script>
+
+
+<figure class="highcharts-figure">
+    <div id="container"></div>
+    <p class="highcharts-description">
+        Compare the performance of a chosen index and a buffer ETF thanks to
+        Highcharts' built-in series compare option, which handles all of the
+        calculations under the hood. Adjust the PERIOD_CAP and PERIOD_BUFFER
+        parameters in the demo, to see how plotLines and plotBands get adjusted
```

Comment:
Use either plain English or `<code>` tags for code keywords. In this case plain English should be okay.
```suggestion
        parameters in the demo, to see how plot lines and plot bands get adjusted
```